### DISCUSSION TITLE
about cache information

### DISCUSSION THREAD

**Discussion by lollita** [winsiderss/systeminformer#1711] [General Discussion]
Why in memory information I not  see about cache as in windows task manager?

**Comment by dmex** [comment]
Task Manager uses `cached`
![image](https://user-images.githubusercontent.com/1306177/236117475-a5a2c8f6-7a92-4276-ada8-18625ccf26c5.png)

System Informer uses `Standby`
![image](https://user-images.githubusercontent.com/1306177/236117494-7b14829b-02c6-44ca-9b05-a31665e7d56b.png)

Standby is the correct name for cached and is shown in the `Memory lists` group.
